cmd/state/stateless/state.go

```diff
@@ -277,10 +293,11 @@ beginTx:
 	}
 
 	processingDone = false
+	i = 0
```

Comment:
You probably don't need this, because `i` is only used to control commits to `localTx` for `lastTimestamps`. If you reset it here, in the worst case you might have slightly unfinished transaction from the previous part of analysis, and not you need to wait more before you commit

Reply:
I did it just to make printing in console more readable - “if digits became small then we started step 2”. Actually I noticed that commits of bolt are very fast - so, we can afford make it more often.